linalg_ops/attention/generate_e2e_attention_tests.py

```diff
@@ -208,20 +262,47 @@ def generate_function(
     func_definition = ""
 
     signature = f"({query_tensor_type}, {key_tensor_type}, {value_tensor_type}, {result_tensor_type}) -> {result_tensor_type}"
+    # print("signature",signature)
+
     import_declaration = f"func.func private @module.{func_name}(%query: !hal.buffer_view, %key: !hal.buffer_view, %value: !hal.buffer_view, %scale: {F32}) -> !hal.buffer_view"
     func_definition = func_definition + (
         f"func.func @{func_name}(%query: {query_tensor_type}, %key: {key_tensor_type}, %value: {value_tensor_type}, %scale: {F32}) -> {result_tensor_type} {{\n"
-        f"  %result0 = tensor.empty(): {result_tensor_type}\n"
-        f"  %scale_f16 = arith.truncf %scale : {F32} to {F16} \n"
+    )
+    if Dynamicity.MIXED == dynamicity:
```

Comment:
style nit (feel free to ignore): I'm used to seeing the variable being checked listed first, so this feel reversed to me
```suggestion
    if dynamicity == Dynamicity.MIXED:
```

Reply:
I will update that; It is better to be consistent.